dom.bs

```diff
@@ -1042,6 +1045,9 @@ are not to be used for anything else. [[!HTML]]
   <a for="event listener">callback</a> will only be invoked once after which the event listener will
   be removed.
 
+  <p>When passed a signal, <var>options</var>'s {{AddEventListenerOptions/signal}} when its
+  [=AbortSignal/aborted flag=] is set the <a for="event listener">callback</a> will be removed.
```

Comment:
This phrasing is a bit awkward. In general this section is web-developer facing summary documentation, so it's OK to depart from the nearby sentences. For example, I might suggest something like

> If an `AbortSignal` is passed for _options_'s `signal`, then the event listener will be removed when signal is aborted.